pl_bolts/models/detection/faster_rcnn/faster_rcnn_module.py

```diff
@@ -107,14 +98,14 @@ def training_step(self, batch, batch_idx):
         # fasterrcnn takes both images and targets for training, returns
         loss_dict = self.model(images, targets)
         loss = sum(loss for loss in loss_dict.values())
-        return {"loss": loss, "log": loss_dict}
+        self.log("loss", loss, prog_bar=True)
 
     def validation_step(self, batch, batch_idx):
         images, targets = batch
         # fasterrcnn takes only images for eval() mode
         outs = self.model(images)
         iou = torch.stack([_evaluate_iou(t, o) for t, o in zip(targets, outs)]).mean()
-        return {"val_iou": iou}
+        self.log("val_iou", iou, prog_bar=True)
```

Comment:
return statement is still needed for the computations in `validation_epoch_end`
```suggestion
        self.log("val_iou", iou, prog_bar=True)
        return {"val_iou2: iou}
```
should fix the tests